types/tuple.hh

```diff
@@ -141,6 +141,24 @@ public:
         boost::range::for_each(range, put);
         return ret;
     }
+    template <typename RangeOf_bytes_opt>  // also accepts bytes_view_opt
+    static managed_bytes build_value_fragmented(RangeOf_bytes_opt&& range) {
```

Comment:
Constraint?

Reply:
This function only exists for a part of this series. It is later replaced with a version which takes fragmented input, and that one has a constraint.